daffodil-core/src/main/scala/org/apache/daffodil/core/runtime1/TermRuntime1Mixin.scala

```diff
@@ -332,24 +339,27 @@ trait TermRuntime1Mixin { self: Term =>
           res.getOrElse(Open(Nil))
         }
       }
-      val res = thisItself match {
-        //
-        // check this case separately first to avoid evaluating
-        // followingLexicalSiblingStreamingUnparserElements unless
-        // we have to.
-        //
-        case Closed(pnes1) => thisItself // most common case - a required element.
-        case _ => {
-          val res: PossibleNextElements =
-            (thisItself, followingLexicalSiblingStreamingUnparserElements) match {
-              case (Open(pnes1), Closed(Nil)) =>
-                thisItself // case of Open(...) followed by end of complex element.
-              case (poss1, Closed(pnes2)) => Closed((poss1.pnes ++ pnes2).distinct)
-              case (poss1, poss2) => Open((poss1.pnes ++ poss2.pnes).distinct)
-            }
-          res
+      // System.err.println("thisItself: " + thisItself)
```

Comment:
Either delete these println, or insert a line ```// keep these println for debug``` at least.

But it would be better to convert these to logging, and make the messages a bit less terce.

I'd like to think that turning on the logging on some reasonably sized test case, one could reason from the output to convince oneself it is working right.

Reply:
They have now been changed to log debug statements.